website/packages/docs/src/pages/pkg-webpack-loader.mdx

```diff
@@ -211,7 +211,6 @@ module.exports = {
 `@compiled/webpack-loader` also accepts the following options. These are not used in the loader itself, but instead they are passed directly to the underlying `@compiled/babel-plugin`.
 
 - `addComponentName`
-- `classHashPrefix`
```

Comment:
This was actually on the wrong place, this says it works with `@compiled/webpack-loader`, but I'm _pretty sure_ it only works with `new CompiledExtractPlugin({…})`.  Could be wrong!